### PR TITLE
client,snappy: use _ for JSON fields

### PR DESCRIPTION
**Pull request by zyga**

### CODE REVIEW

**Comment by chipaca** [comment]
Don't forget `rest.md` and `meta.md`

**Comment by chipaca** [comment]
+1 once it's fixed in the docs

File: snappy/snap_yaml.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ type AppYaml struct {
 	Description string          `yaml:"description,omitempty" json:"description,omitempty"`
 	Stop        string          `yaml:"stop,omitempty"`
 	PostStop    string          `yaml:"poststop,omitempty"`
-	StopTimeout timeout.Timeout `yaml:"stop-timeout,omitempty"`
+	StopTimeout timeout.Timeout `yaml:"stop_timeout,omitempty"`
```
Comment (pedronis) [inline snappy/snap_yaml.go]: this a yaml, don't know where it goes?

Comment (pedronis) [inline snappy/snap_yaml.go]: actually, it seems this is json/yaml, so it would need double defs, missing tests?

**Comment by niemeyer** [comment]
Please don't smash changes. Clicking on revision 79df9c2 should show what you changed, not the whole thing again.

**Comment by mvo5** [comment]
Close per zygas request

**Pull request closed without merging**